src/main/java/frc/robot/Robot.java

```diff
@@ -4,68 +4,258 @@
 
 package frc.robot;
 
+import com.revrobotics.CANSparkMax;
```

Comment:
I don't think you see a class with 23 imports in your everyday life (import the whole autonomous, subsystems and services package using `.*`)